workloads/router-perf-v2/env.sh

```diff
@@ -10,8 +10,8 @@ export ES_INDEX=${ES_INDEX:-router-test-results}
 NUM_NODES=$(oc get node -l node-role.kubernetes.io/worker,node-role.kubernetes.io/workload!=,node-role.kubernetes.io/infra!= --no-headers | grep -cw Ready)
 LARGE_SCALE_THRESHOLD=${LARGE_SCALE_THRESHOLD:-24}
 METADATA_COLLECTION=${METADATA_COLLECTION:-true}
-ENGINE=${ENGINE:-podman}
-KUBE_BURNER_RELEASE_URL=${KUBE_BURNER_RELEASE_URL:-https://github.com/cloud-bulldozer/kube-burner/releases/download/v0.11/kube-burner-0.11-Linux-x86_64.tar.gz}
+ENGINE=${ENGINE:-local}
+KUBE_BURNER_RELEASE_URL=${KUBE_BURNER_RELEASE_URL:-https://github.com/cloud-bulldozer/kube-burner/releases/download/v0.15.4/kube-burner-0.15.4-Linux-x86_64.tar.gz}
```

Comment:
Instead of updating this every time we release do we want to just set it to latest via something like
```
$ curl -s "https://api.github.com/repos/cloud-bulldozer/kube-burner/releases/latest" | jq -r '.assets[] | .browser_download_url' | grep Linux-x86_64
```

Reply:
I personally prefer to stick with a fixed version, that way we can push changes to kube-burner safely

Reply:
ack